spec/lib/traffic_source_parser/parser/utmz_parser_spec.rb

```diff
@@ -54,6 +54,43 @@
 
       end
 
+      context 'when value is from referral' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.